kotti/util.py

```diff
@@ -175,12 +175,13 @@ def get_visible_children(self, context, request):
 class Link(LinkBase):
     template = 'kotti:templates/edit/el-link.pt'
 
-    def __init__(self, name, title=None, predicate=None):
+    def __init__(self, name, title=None, predicate=None, target=None):
```

Comment:
I think this should be ``target='_self'``.  Empty target attributes are not valid IIRC.